src/app/organizations/organization/organization.component.ts

```diff
@@ -35,14 +36,27 @@ export class OrganizationComponent implements OnInit {
   organization$: Observable<Organization>;
   loading$: Observable<boolean>;
   canEdit$: Observable<boolean>;
-  constructor(private organizationQuery: OrganizationQuery, private organizationService: OrganizationService, private matDialog: MatDialog
-  ) { }
+  gravatarUrl: string;
+  defaultImgUrl = 'https://via.placeholder.com/150';
+  constructor(private organizationQuery: OrganizationQuery, private organizationService: OrganizationService,
+              private matDialog: MatDialog) { }
+
+  protected ngUnsubscribe: Subject<{}> = new Subject();
 
   ngOnInit() {
     this.loading$ = this.organizationQuery.loading$;
     this.canEdit$ = this.organizationQuery.canEdit$;
     this.organizationService.updateOrganizationFromNameORID();
     this.organization$ = this.organizationQuery.organization$;
+    this.organization$.pipe(takeUntil(this.ngUnsubscribe)).subscribe(o => {
+      if (o) {
```

Comment:
`if (o && o.avatarUrl)` -- if either is false you want to set gravatarUrl to null

Reply:
changed